jobfunnel/backend/scrapers/indeed.py

```diff
@@ -419,4 +420,66 @@ def _get_num_search_result_pages(self, search_url: str, max_pages=0) -> int:
         elif number_of_pages < max_pages:
             return number_of_pages
         else:
-            return max_pages
\ No newline at end of file
+            return max_pages
+
+
+class IndeedScraperGEGer(BaseIndeedScraper, BaseGEGerScraper):
+     '''Scrapes jobos from indeed.de
+     '''
+     def _get_search_url(self, method: Optional[str] = 'get') -> str:
```

Comment:
I think you can remove `_get_search_url` and `_get_num_search_result_pages` here as these are defined by `BaseIndeedScraper`